discord/types/role.py

```diff
@@ -30,11 +30,18 @@
 from .snowflake import Snowflake
 
 
+class RoleColours(TypedDict):
+    primary_color: int
+    secondary_color: int | None
+    tertiary_color: int | None
+
+
 class Role(TypedDict):
     tags: NotRequired[RoleTags]
     id: Snowflake
     name: str
     color: int
```

Comment:
ultimately color will be deprecated by discord in the future
might be worth warning already

Reply:
Yeah good idea